.github/workflows/on_push.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         with:
           registry: ${{ env.CONTAINER_REGISTRY }}
           username: ${{ env.CONTAINER_REGISTRY_USER }}
-          password: ${{ secrets.CONTAINER_REGISTRY_PASSWORD }}
+          password: ${{ env.CONTAINER_REGISTRY_PASSWORD }}
```

Comment:
I don't know, to me this seems like a mighty bad idea? What was the reason for using env over secret here?

Reply:
Duplication. As `CONTAINER_REGISTRY_PASSWORD` is stored to env at the beginning (https://github.com/andersinno/kolga/pull/9/files#diff-53dbc06610f897f66670a951aca3d54295c4a0fa85d7869295330aa05265a761R9), I don't see point of duplicating it's reading from secret. 

With the change it's also similar to registry and username variables.